controllers/article-controllers.js

```diff
@@ -16,4 +16,16 @@ exports.getArticles = (req,res,next) => {
   selectArticles().then((result) => {
     res.status(200).send({articles: result})
   })
+}
+
+exports.getArticleComments = (req, res, next) => {
+ const { article_id } = req.params
+ const promise1 = selectArticleById(article_id)
+ const promise2 = selectArticleComments(article_id)
+
+ const dbPromises = [promise1, promise2]
```

Comment:
this should be done in one query, think about how we could join the comments to the articles?